pkg/registry/common/grpcmetadata/nse_client.go

```diff
@@ -39,7 +39,7 @@ func NewNetworkServiceEndpointRegistryClient() registry.NetworkServiceEndpointRe
 func (c *grpcMetadataNSEClient) Register(ctx context.Context, nse *registry.NetworkServiceEndpoint, opts ...grpc.CallOption) (*registry.NetworkServiceEndpoint, error) {
 	path, err := PathFromContext(ctx)
 	if err != nil {
-		return nil, err
+		path = &Path{}
 	}
```

Comment:
Should we rework api https://github.com/networkservicemesh/sdk/blob/a64652cbc2d5acb666014d5d465d2a7ed615548d/pkg/registry/common/grpcmetadata/common.go#L34-L40

and don't return the error instead return the empty path?